tests/rest/ssh_dashboard.json

```diff
@@ -1,285 +1,16 @@
 {
-  "title": "Your Basic Dashboard",
-  "services": {
-    "query": {
-      "list": {
-        "0": {
-          "query": "*",
-          "alias": "",
-          "color": "#7EB26D",
-          "id": 0,
-          "pin": false,
-          "type": "lucene",
-          "enable": true
-        }
-      },
-      "ids": [
-        0
-      ]
-    },
-    "filter": {
-      "list": {},
-      "ids": []
+  "dashboard": {
+    "title": "Your Basic Dashboard",
+    "hits": 0,
```

Comment:
hits should be removed as this also causes problems, kibana keeps track of hits on it's own and will autopopulate this field.

Reply:
Good catch, removed!